game-core/src/main/java/org/triplea/game/server/HeadlessGameServer.java

```diff
@@ -66,7 +66,7 @@
   private final List<Runnable> shutdownListeners = Arrays.asList(
       lobbyWatcherResetupThread::shutdown,
       () -> Optional.ofNullable(game).ifPresent(ServerGame::stopGame),
-      () -> setupPanelModel.getPanel().cancel());
+      () -> Optional.ofNullable(setupPanelModel.getPanel()).ifPresent(HeadlessServerSetup::cancel));
```

Comment:
btw. This fixes a `NullPointerException` I encountered when shutting down the bot.